tools/oldest_constraints.txt

```diff
@@ -26,15 +26,6 @@ zope.component==4.1.0
 zope.event==4.0.3
 zope.interface==4.0.5
 
-# Debian Jessie Backports constraints
-PyICU==1.8
```

Comment:
The full test suite initially failed here (you can see the log [here](https://gist.github.com/71c2b318f29ffc65cb47c7f54727110e)), but it was in our unpinned tests and was a spurious failure.

I agree with Erica's comment though. We need to do something else here.

For what it's worth, I personally think we can either:

1. Leave those dependencies pinned to those versions, but update the comment referencing Debian Jessie to say that since Debian Jessie has reached it's end-of-life, we can probably update these packages to a newer version if needed.
2. Take the time now to figure out the new oldest version of those packages we need to support is and update the pinned version in this file accordingly. The way to check this would be to look at the versions of these packages found in Debian 9, CentOS 7 with EPEL 7, and Ubuntu 16.04 and update the pinning to the oldest version found and move it to the correct section of this file to document where that version constraint is coming from.

Reply:
I think going with option 1 is fine.

Reply:
But if we do that, the comment should also make sure to include the places to check mentioned in option 2, because we will want to do that instead of just deleting things willy-nilly in the hypothetical future.

Reply:
I wasn't sure I understood initially but I checked with Brad and I think I made the desired change.  I'm not as up on the context as you guys are, so please let me know if it needs something more.